app/models/virtue.rb

```diff
@@ -1,3 +1,7 @@
 class Virtue < ApplicationRecord
   belongs_to :character_class
+  def self.select_values(character_class)
```

Comment:
Again, we should task a model with a responsibility of knowing how to render itself somewhere. Please think about a better place of placing this code. Consider a presenter/decorator pattern: https://github.com/drapergem/draper